properties/math.py

```diff
@@ -12,7 +12,7 @@
 from .utils import ValidationError
 
 TYPE_MAPPINGS = {
-    int: 'i',
+    int: 'ui',
```

Comment:
I guess I don't quite follow here... why doesn't this break signed `int`s?

Reply:
Unsigned ints have `kind='u'`, signed have `kind='i'` - so the check ends up as `kind in 'ui'` or if you do something like `..., dtype=(int, float, bool)` the check will end up as `kind in 'uifb'`. Make sense?

Reply:
Yep, thanks :)